lib/validate.js

```diff
@@ -226,6 +226,53 @@ function assertGelfAddress(arg, name) {
     }
 }
 
+/*
+ * Docker docs: Supported standard values are:
+ * bridge, host, none, and container:<name|id>,
+ * Any other value is taken as a custom network’s
+ * name or id to which the provisioning container
+ * should connect.
+ *
+ * We support: bridge, other.
+ */
+function assertNetworkMode(networkmode, name) {
+    if (networkmode === 'bridge') {
+        return;
+    }
+
+    if (networkmode === 'host' || networkmode === 'none') {
+        throw new errors.ValidationError(fmt('NetworkMode %s is not supported',
```

Comment:
Would [NotImplementedError](https://github.com/joyent/sdc-docker/blob/master/lib/errors.js#L256-L268) be better suited for such errors?